src/errors.rs

```diff
@@ -28,6 +28,7 @@ pub enum Error {
     RedisError(RedisError),
     /// Errors for using incomplete block(like no proof block) as usual block.
     IncompleteBlock,
+    InvalidFederation(String),
```

Comment:
I prefer to define as `InvalidFederation(&'static str)` because we can remove 'to_string()' in initializing it.


```
InvalidFederation("The nodevss...".to_string())
```
↓
```
InvalidFederation("The nodevss...")
```